delta/sdk/src/test/resources/resolvers/database/resolver-cross-project-created-1.json

```diff
@@ -14,6 +14,8 @@
   },
   "value": {
     "@type": "CrossProjectValue",
+    "name": null,
```

Comment:
We should not encode the nulls.
We should have at least one of the examples of serialization to have these values populated

Reply:
Added test with resolvers that have non null name/description fields.